src/backend/_scss/application.scss

```diff
@@ -1072,13 +1072,9 @@ a {
   .speaker-description {
     float: left;
     max-height: 262.5px;
-    overflow: hidden;
+    overflow: auto;
```

Comment:
@geekyd Overflow : auto will introduce a `scrollbar` which we don't want initially. We want scrollbar only on hover.